src/user-guide/writing-workflows/suicide-triggers.rst

```diff
@@ -0,0 +1,116 @@
+
+.. TAKEN OUT OF THE USER GUIDE FOR CYLC 8
+   We made need to keep this as a reference for rare remaining use cases
+
+.. _SuicideTriggers:
+
+Suicide Triggers
+^^^^^^^^^^^^^^^^
+
+.. tutorial:: Suicide Trigger Tutorial <tut-cylc-suicide-triggers>
+
+Suicide triggers take tasks out of the workflow. This can be used for automated
+failure recovery. The following :cylc:conf:`flow.cylc` listing and accompanying
+:term:`graph` show how to define a chain of failure recovery tasks that trigger
+if they're needed but otherwise remove themselves from the workflow (you can run
+the *AutoRecover.async* example workflow to see how this works). The dashed graph
+edges ending in solid dots indicate suicide triggers, and the open arrowheads
+indicate conditional triggers as usual.
+
+.. Need to use a 'container' directive to get centered image with
+   left-aligned caption (as required for code block text).
+
+.. container:: twocol
+
+   .. container:: caption
+
+      *Automated failure recovery via suicide triggers*
+
+      .. code-block:: cylc
+
+          [meta]
+              title = automated failure recovery
+              description = """
+                  Model task failure triggers diagnosis
+                  and recovery tasks, which take themselves
+                  out of the workflow if model succeeds. Model
+                  post processing triggers off model OR
+                  recovery tasks.
+              """
+          [scheduling]
+              [[graph]]
+                  R1 = """
+                      pre => model
+                      model:fail => diagnose => recover
+                      model => !diagnose & !recover
+                      model | recover => post
+                  """
+          [runtime]
+              [[model]]
+                  # UNCOMMENT TO TEST FAILURE:
+                  # script = /bin/false
+
+   .. container:: image
+
+      .. _fig-suicide:
+
+      .. figure:: ../../img/suicide.png
+         :align: center
```

Comment:
Dump in a `digraph`, they're easier to maintain. Can raid the old suicide trigger tutorial.

Reply:
(I've completed gutted the suicide trigger section, most of it is not needed anymore)